chromium_src/chrome/browser/unexpire_flags.cc

```diff
@@ -23,6 +23,13 @@ bool IsFlagExpired(const flags_ui::FlagsStorage* storage,
     return true;
   }
 
+  // Enable playlist feature only for nightly/development.
+  if (base::LowerCaseEqualsASCII("playlist", internal_name) &&
```

Comment:
I'm fairly certain this is not the best way to do this. @iefremov ?

Reply:
why not we just add a function `IsPlaylistEnabled()` and check (1) feature (2) channel inside it?

Reply:
The only reason is to make playlist flag visible only for nightly channel.

Reply:
then this code makes sense. Though personally I'm not sure whether we should hide it - it anyways can be overridden via the command line